src/components/BoardContent/BoardContent.js

```diff
@@ -104,6 +104,30 @@ function BoardContent() {
 		toggleOpenNewColumnForm();
 	};
 
+	const onUpdateColumn = (newColumnToUpdate) => {
+		const columnIdToUpdate = newColumnToUpdate.id;
+		let newColumns = [...columns];
+		const columnIndexToUpdate = newColumns.findIndex(
+			(i) => i.id === columnIdToUpdate
+		);
+
+		if (newColumnToUpdate._destroy) {
+			// remove column
+			newColumns.splice(columnIndexToUpdate, 1);
+		} else {
+			// update column
+			newColumns.splice(columnIndexToUpdate, 1, newColumnToUpdate);
+		}
+
+		// Change columns and board state
+		let newBoard = { ...board };
```

Comment:
Need to refactor code to reuse those lines of code (update board and update columns)